src/products/model.ts

```diff
@@ -0,0 +1,17 @@
+import {productsDBConnection} from './db'
+import {DataTypes} from "sequelize";
+import {ProductInstance} from "./types";
+
+const ProductModel = productsDBConnection.define<ProductInstance>("products", {
+    id: {
+        type: DataTypes.INTEGER,
+        primaryKey: true,
+        autoIncrement: true
+    },
+    title: DataTypes.STRING,
+    description: DataTypes.STRING,
+    price: DataTypes.INTEGER,
```

Comment:
```suggestion
    price: {
      type: DataTypes.INTEGER.
      allowNull: false,
      validate: {
        isFloat: true
      }
    },
    
    
```

also add `allowNull: false` to all non primary key fields.